sccp/figures/figure3a.py

```diff
@@ -27,7 +27,7 @@ def makeFigure():
     CompW = allP @ factors[1]
 
     # UMAP dimension reduction
-    ump = umap.UMAP(random_state=1).fit_transform(allP)
+    ump = umap.UMAP(random_state=1).fit_transform(CompW)
```

Comment:
The UMAP positions should be based only on the projections, not B * projections which are the factor effects.